chalice/app.py

```diff
@@ -603,6 +608,11 @@ def __call__(self, event, context):
                                        event['requestContext'],
                                        event['stageVariables'],
                                        event.get('isBase64Encoded', False))
+        # We're getting the CORS headers before validation to be able to
+        # output desired headers with
+        cors_headers = dict()
```

Comment:
Can we change this to `None` instead?

If we do this the `None` default of the headers argument in the `error_response` function will never be useful since its always a `dict`, empty or not. The expected input to the `Response` object if there are no headers is `None` so I would like to keep that consistent.